### PR TITLE
Dummy routes

### PR DESCRIPTION
**Pull request by Alesfatalis**

### CODE REVIEW

File: api/controllers/NewsController.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+let newsCategories = require('../../json/newsCategories');
+
+module.exports = {
+    news: function (req, res) {
+        res.json(newsData);
+    },
+    categories: function (req, res) {
+        res.json(newsCategories)
+    }
+};
```
Comment (gitBear11) [inline api/controllers/NewsController.js]: add new line

File: api/controllers/CanteenController.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ *
+ * @description :: Actions to deliver menu data.
+ */
+let menuData = require('../../json/canteen');
+
+module.exports = {
+    menu: function (req, res) {
+        res.json(menuData)
+    }
+};
```
Comment (gitBear11) [inline api/controllers/CanteenController.js]: add new line

File: api/controllers/NewsController.js (hunk 1 above)
Comment (Alesfatalis) [reply api/controllers/NewsController.js]: There is already a new line

File: api/controllers/CanteenController.js (hunk 2 above)
Comment (Alesfatalis) [reply api/controllers/CanteenController.js]: There is already a new line

**Pull request merged**